server/public/model/config.go

```diff
@@ -4591,6 +4679,46 @@ func (s *DataRetentionSettings) isValid() *AppError {
 	return nil
 }
 
+func (s *AutoTranslationSettings) isValid() *AppError {
+	if s.Enable != nil && *s.Enable {
```

Comment:
Can we switch this if so that we don't need to indent everything? So something like
```go
if s.Enable == nil || !*s.Enable {
    return nil
}
// rest of the validation
```